src/pages/CustomerInvoicesPage.js

```diff
@@ -61,30 +49,41 @@ export const CustomerInvoicesPage = ({
     dataState,
     sortBy,
     isAscending,
-    columns,
     modalKey,
     hasSelection,
     keyExtractor,
     searchTerm,
+    columns,
```

Comment:
in the above `initializer`, the `filterKeys` should also have `serialNumber`. Also naming inconsistency - whats better `initializer` or `stateInitialiser` 🤷‍♀

Reply:
I prefer `stateInitialiser` and not just because it is using an 's' rather than a 'z'
The initialisation is for the state, and I think not being specific leads me to think that it is initialising the component.